terraform/main.tf

```diff
@@ -206,7 +206,7 @@ data "aws_iam_policy_document" "read_datadog_api_key_secret" {
 
 resource "aws_ses_receipt_rule" "ffis_ingest" {
   depends_on = [
-    aws_iam_policy.ses_source_data_s3_access
+    aws_s3_bucket_policy.ses_source_data_s3_access_policy
   ]
```

Comment:
Is this `depends_on` still necessary?

Reply:
I'm going to tentatively say yes.  Again, I'm not sure how this all fits into Terraform's dependency graph so, just by eyeballing it, I dont see anything that's going to stop the `aws_ses_receipt_rule` from being created before the policy is applied to the bucket.  We can try removing it but I think this will error again complaining about being unable to write to the bucket.